src/shared/util/spawn.js

```diff
@@ -13,14 +13,18 @@ specific language governing permissions and limitations under the License.
 import cp from 'child_process';
 
 import colors from 'colour';
+import { IS_PACKAGED_BUILD } from '../build-info';
 
 export const spawn = (command, main, args, { logger }, options = {}) => new Promise((resolve, reject) => {
   logger.log('Spawning',
     colors.cyan(command),
     colors.blue(main),
     colors.yellow(args.join(' ')));
 
-  const child = cp.spawn(command, [main, ...args], { stdio: 'inherit', ...options });
+  const stdio = process.platform === 'win32' && IS_PACKAGED_BUILD
+    ? 'ignore'
+    : 'inherit';
```

Comment:
So, my theory there (and I was also thinking of moving the `+ '.cmd'` code into this function too) was that it would be nice if `spawn` handled spawning programs, and didn't force all the people writing the various functions that called this to have to know the difference between Windows and macOS and packaged builds and not.  I figured it would be easier to fix these things once, here, instead of constantly running into places we forgot to change.  😉